bblfsh-web/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 name: bblfsh-web
-version: 0.7.0
```

Comment:
Since this adds new features, it is not a patch version. Let's make this a minor version

Reply:
Changed to 0.8.0 then